jail-buddy.lic

```diff
@@ -54,7 +54,12 @@ def get_out_of_jail(old_loc)
   Script.running.find_all { |s| !s.paused? && !s.no_pause_all }.each(&:pause)
   stop_script('go2') if Script.running?('go2')
   echo('** DON\'T PANIC JAIL BUDDY HAS YOUR BACK! **')
+  fput('stand') if not standing?
+  start_script('outdoorsmanship',['10 "dust bunny"'])
+  
   waitfor('PLEAD INNOCENT or PLEAD GUILTY')
+  stop_script('outdoorsmanship') if Script.running?('outdoorsmanship')
```

Comment:
❓  Do we need a `waitrt?` after stopping the script?

Reply:
I don't think so.  I ended up in the 'Pleading' room in RT during testing and 'PLEAD INNOCENT' waited for RT to end before pleading

Reply:
Sounds good